pkg/controller/perconaservermongodb/psmdb_controller.go

```diff
@@ -150,6 +151,15 @@ func (r *ReconcilePerconaServerMongoDB) Reconcile(request reconcile.Request) (re
 		RequeueAfter: r.reconcileIn,
 	}
 
+	// PerconaServerMongoDB object is also accessed and changed by a version service's corn job (that run concurrently)
```

Comment:
`corn job` -> `cron job`
`that run` -> `that runs`

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/393/commits/8b8c5a92a3dcfedeb74621c8f7493e332ed9bf56